src/api/app/jobs/send_event_emails.rb

```diff
@@ -16,16 +17,29 @@ def perform
       subscribers = event.subscribers
       next if subscribers.empty?
       EventMailer.event(subscribers, event).deliver_now
-
-      event.subscriptions.each do |subscription|
-        Notifications::RssFeedItem.create(
-          subscriber: subscription.subscriber,
-          event_type: event.eventtype,
-          event_payload: event.payload,
-          subscription_receiver_role: subscription.receiver_role
-        )
-      end
+      create_rss_notification(event, renderer)
     end
     true
   end
+
+  private
+
+  def create_rss_notification(event, renderer)
+    description = renderer.render(
+      template: "event_mailer/#{event.template_name}",
+      layout: false,
+      format: :txt,
+      assigns: { e: event.expanded_payload, host: ::Configuration.obs_url, configuration: ::Configuration.first })
+
+    event.subscriptions.each do |subscription|
+      Notifications::RssFeedItem.create(
+        subscriber: subscription.subscriber,
+        event_type: event.eventtype,
+        event_payload: event.payload,
+        title: event.subject,
+        description: description,
```

Comment:
Talking about what is needed to store... just the payload, right. 

But in the other hand every time  we are going to show the feed item we have to instantiate a class by the name stored in the event_type, then call subject on it and also expand the payload for rendering the view (in some cases it means finding the User object)...

When we talked I was for just having that payload and rendering it after, but now I'm not that sure about what is the best approach.